config/locales/cfp.fr.yml

```diff
@@ -0,0 +1,108 @@
+    contact_headline: "Contact"
+    create_event_headline: "Créer un nouvel évènement"
+    current_password_hint: "nous avons besoin de votre mot de passe pour confirmer ces changements."
+    #deadline_description: "Veuillez soumettre votre proposition avant"
+    description_hint: "Une description plus longue de vous. Utilisez cet espace pour donner plus de détails que dans l'extrait. (HTML)"
+    detailed_description: "Description détaillée"
+    edit_account: "Editer votre compte"
+    edit_availability: "Editer votre disponibilité"
+    edit_availability_description: "Veuillez nous faire savoir vos disponibilités pour les évènements que vous proposez. Nous ferons de notre mieux pour en tenir compte lors de la création du planning."
+    edit_availability_headline: "Edition de la disponibilité"
+    edit_event_headline: "Edition d'évènement"
+    edit_profile: "Editer votre profil personnel"
+    email_hint: "Veuillez entrer l'adresse emails que nous utiliserons comme moyen principal pour vous contacter."
+    email_public_hint: "Si vous le souhaitez, cette adresse email peut être publiée dans notre programme public."
+    error_confirming: "Une erreur est survenue lors de la confirmation de votre compte. Veuillez vérifier que vous avez collé le lien complet depuis l'email, et que vous n'avez pas déjà activé votre compte."
+    error_sending_confirmation_instructions: "Impossible d'envoyer les instructions de confirmation. Veuillez vérifier que vous avez entré une adresse email correcte et que votre compte n'est pas déjà créé."
+    event_abstract_hint: "Un petit parapgraphe décrivant succintement l'évènement. (HTML)"
+    submission_note_hint: "Visible pour les admins et les utilisateurs, contient des informations additionnelles."
+    event_confirmed: "Évènement confirmé"
+    event_created_notice: "L'évènement a été créé avec succès."
+    event_description_hint: "Une description plus longue de l'évènement. Utilisez cet espace pour donner plus de détails. L'extrait et la description seront visibles sur le site de la conférence. (HTML)"
+    event_time_slots_hint: "Durée de votre évènement, en heures et minutes. Pour des talks standards, laissez les paramètres par défaut."
+    event_updated_notice: "L'évènenement a été mis à jour avec succès."
+    event_withdrawn_notice: "Votre proposition a été retirée."
+    events_submitted: "Évènements que vous avez déjà proposés :"
+    forgot_password_button: "Demander un nouveau mot de passe"
+    forgot_password_description: "Vous êtes déjà venu·e mais vous ne vous souvenez plus de votre mot de passe ? C'est embêtant, mais vous pouvez"
+    forgot_password_headline: "Mot de passe oublié?"
+    hard_deadline_over: "Désolé mais la date limite de soumission est dépassée."
+    im_accounts: "Comptes de messagerie instantanée"
+    info_url_description_html: "Pas sûr·e de ce que vous devez envoyer ? lisez l'Appel à paricipation ici : %{link}"
```

Comment:
Missing capitalization "Lisez"

:+1: Inclusive language

Reply:
What about that ? "Rejette cet évènement. Le(s) orateur·trices(s) seront/sera notifié·e(s) de ce rejet par e-mail."

Reply:
Not sure singular add value… "Rejette cet évènement. Les orateurs·trices seront notifié·e·s de ce rejet par e-mail." LGTM (but I can't see it in this scope BTW)

Reply:
I'll go with "Les orateurs·trices seront notifiés·es de ce rejet par e-mail.", this is more global for a button hint that rejects events.